package.json

```diff
@@ -34,6 +34,7 @@
     "stage": "NODE_ENV=production npm run production && gcloud app deploy --project=chrome-apps-doc-staging-2020 --quiet",
     "start": "node ./server",
     "test:watch": "ava --watch",
+    "pretest": "cd tests/e2e/fixtures && eleventy",
```

Comment:
is it used somewhere? Or is it intended to be used manually?

Reply:
You can prefix any script name with `pre` or `post` to make them run with the script with the unprefixed name. So this would automatically run when `npm run test` is called. Also see https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts